src/Libraries/Microsoft.Extensions.Caching.Hybrid/Internal/DefaultHybridCache.cs

```diff
@@ -20,8 +21,11 @@ namespace Microsoft.Extensions.Caching.Hybrid.Internal;
 /// <summary>
 /// The inbuilt implementation of <see cref="HybridCache"/>, as registered via <see cref="HybridCacheServiceExtensions.AddHybridCache(IServiceCollection)"/>.
 /// </summary>
+[SkipLocalsInit]
```

Comment:
why `SkipLocalsInit` exists on specific types only? Not on module?

Reply:
I'd be fine with either, but I don't really *need* it globally; least surprises, etc